{
  "id": "CVE-2020-36271",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2022-08-22T16:15:09.067",
  "lastModified": "2022-08-22T16:15:09.067",
  "vulnStatus": "Rejected",
  "descriptions": [
    {
      "lang": "en",
      "value": "** REJECT ** DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Reason: This candidate was withdrawn by its CNA. Further investigation showed that it was not a security issue. Notes: none."
    }
  ],
  "metrics": {},
  "references": []
}